<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>FileDialog</class>
 <widget class="QDialog" name="FileDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="5" column="1" colspan="4" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>all produced files</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QPushButton" name="clearButton" >
     <property name="text" >
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QPushButton" name="moveFilesButton" >
     <property name="text" >
      <string>Move</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>58</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>kB in &quot;created&quot; directory.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4" >
    <widget class="QLabel" name="label_3_2" >
     <property name="text" >
      <string>all files from &quot;created&quot; into &quot;movedFiles&quot; directory.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4" >
    <widget class="QLabel" name="label_4_2" >
     <property name="text" >
      <string>all files from &quot;created&quot; into &quot;copiedFiles&quot; directory.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="copyFilesButton" >
     <property name="text" >
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="moveDirButton" >
     <property name="text" >
      <string>Move</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="copyDirButton" >
     <property name="text" >
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>&quot;created&quot; into &quot;copied&quot; directory.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>&quot;created&quot; into &quot;moved&quot; directory.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QPushButton" name="createButton" >
     <property name="text" >
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSpinBox" name="filesSpinBox" >
     <property name="maximum" >
      <number>1000</number>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>files, each of size</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QSpinBox" name="sizeSpinBox" >
     <property name="maximum" >
      <number>10000</number>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>167</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton_2_2" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="6" row="1" column="5" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Copy Flags</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="makeLinksCheckBox" >
        <property name="text" >
         <string>Make Links</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="followLinksCheckBox" >
        <property name="text" >
         <string>Follow Links</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="forceCheckBox" >
        <property name="text" >
         <string>Force</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>46</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton_2_2</sender>
   <signal>clicked()</signal>
   <receiver>FileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>351</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel" >
     <x>310</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
